package/yast2-auth-server.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Jul 27 00:50:39 UTC 2022 - William Brown <william.brown@suse.com>
+
+- Version 4.5.1
+- Remove kerberos ldap database support
```

Comment:
```suggestion
- Remove kerberos ldap database support (gh#yast/yast-auth-server#73)
- 4.5.1
```

Reply:
I think proposed change above should be enough. You can just accept the suggestion or add a new commit by yourself. Sorry for the noisy review :wink: